.kokoro/common.cfg

```diff
@@ -0,0 +1,24 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```

Comment:
Nit: will the `.kokoro` file be included in `npm` installs of this library?

(Ideally it wouldn't be, but not the end of the world if it is.)

Reply:
I added `files` field in package.json, so only the source will be included.